lib/shelley/bench/Latency.hs

```diff
@@ -203,15 +204,16 @@ walletApiBench capture ctx = do
     nFixtureWallet n = do
         wal1 : wal2 : _ <- replicateM n (fixtureWallet ctx)
         walMA <- fixtureMultiAssetWallet ctx
-        pure (wal1, wal2, walMA)
+        walMAForMigration <- fixtureMultiAssetWallet ctx
+        pure (wal1, wal2, walMA, walMAForMigration)
```

Comment:
Hi @piotr-iohk! This is just a suggestion, but perhaps we could use a name that makes it obvious that it's a source wallet to be migrated:

```suggestion
        pure (wal1, wal2, walMA, maWalletToMigrate)
```

What do you think about this name?

("For migration" could imply that it's a target wallet.)